sdgym/benchmark.py

```diff
@@ -133,12 +113,10 @@ def _compute_scores(metrics, real_data, synthetic_data, metadata, output, comput
             normalized_score = None
             start = datetime.utcnow()
             try:
-                LOGGER.info('Computing %s on dataset %s', metric_name, metadata._metadata['name'])
```

Comment:
I think we can keep the logs

Reply:
I deleted them because we don't have the parameters anymore in most of them. I can try to fix that though.

Reply:
yeah let's try to get them or something close. I think we could use more logging in our libraries lol